src/org/zaproxy/zap/extension/alert/ExtensionAlert.java

```diff
@@ -74,6 +75,7 @@
 		SessionChangedListener, XmlReporterExtension, OptionsChangedListener {
 
     public static final String NAME = "ExtensionAlert";
+    private static final Logger logger = Logger.getLogger(ExtensionAlert.class);

```

Comment:
Const? (Full caps)

Reply:
Didn't change the case as that would require update a lot of places.

Reply:
Ok